website/src/theme/DocItem/TOC/Desktop/index.js

```diff
@@ -11,7 +11,7 @@ export default function DesktopWrapper(props) {
 
   return shouldRenderAMASidebar ? (
     <div className="with-ama-sidebar">
-      <AMASidebar />
+      <AMASidebar className="ama-sidebar" />
```

Comment:
className is already applied on `<AMASection className="ama-sidebar" />` in `<AMASidebar />` and `className` here is not pass through props so it won't have any effect. 

```suggestion
      <AMASidebar />
```